content/articles/genie-slide-in-jelly-and-scale-pop-up-notification-effects/index.md

```diff
@@ -0,0 +1,1107 @@
+
+### Add the growl effect to the notification
+In the `growl-notification-style.css` file, add growl effects. Find the code and some inline commentary of the growl effects CSS code [here](https://github.com/prograte/Genie-Slide-in-Jelly-and-scale-Pop-up-Notification-effects/blob/main/css/growl-notification-style.css).
+
+- Added growl styling that will be shared with all the types of growl effects
+- Added individual effects for each of the growl effect sets. This starts with the scale effect. The _scale_ effect copies its name. It expands from a small size to a larger size. It produces a notification with the animation shown in
+  the GIF below:
+
+![Scale effect gif](engineering-education/genie-slide-in-jelly-and-scale-pop-up-notification-effects/scale-effect-gif.gif "Scale effect gif")
+
+- Added the _Jelly_ animation effect. Some part of it is generated from [Bouncejs.com](http://bouncejs.com/).
+
+  > [Bouncejs.com](http://bouncejs.com/) is a site used to generate awesome CSS3 animations for a website. Click on the site to check out some presets available or create animations that are tailored for the site.
+
+Head over to the `growl-effects-jelly.html` either by opening it in a browser or clicking the `Jelly` button on the
+webpage. Click on `SHOW NOTIFICATION` to see the animation. The animation looks like that in the GIF below:
+
+![Jelly effect gif](engineering-education/genie-slide-in-jelly-and-scale-pop-up-notification-effects/jelly-effect-gif.gif "Jelly effect gif")
+
+- Added the slide-in animation to the project. Open the `growl-effects-slide-in.html` by clicking on the `Slide in`
+  button. This file has the notification style set to slide hence it will implement the `slide in` effect. The effect is
+  displayed in the GIF shown below:
+
+![Slide-in effect gif](engineering-education/genie-slide-in-jelly-and-scale-pop-up-notification-effects/slide-in-effect-gif.gif "Slide-in effect gif")
+
+- Finally, in the file, add the Fly Up effect. This effect can be seen on the `growl-effects-genie.html` file accessed by clicking on the  `Genie` button. The code for this is shown below:
+  ![genie effect gif](engineering-education/genie-slide-in-jelly-and-scale-pop-up-notification-effects/genie-effect-gif.gif "genie effect gif")
+
+### Normalize the webpages
+Copy and paste the content of the `normalization.css` file
+from [here](https://github.com/prograte/Genie-Slide-in-Jelly-and-scale-Pop-up-Notification-effects/blob/main/css/normalization.css)
+. Do this by copy-pasting the code directly or viewing it as a _RAW_ file before copy-pasting it.
+
+The code in the file styles the webpage elements to look well. It is additional formatting to make the webpage look
+appealing to other browsers.
+In case of any issues, find the code for the article in [this](https://github.com/prograte/Genie-Slide-in-Jelly-and-scale-Pop-up-Notification-effects) repository.
+Modify it to create desired outputs for projects.
```

Comment:
```suggestion
The code in the file styles the webpage elements to look well. It is additional formatting to make the webpage look appealing to other browsers.

In case of any issues, find the code for the article in [this repository](https://github.com/prograte/Genie-Slide-in-Jelly-and-scale-Pop-up-Notification-effects). Modify it to create desired outputs for projects.
```